x-pack/plugins/maps/public/shared/layers/sources/es_search_source/es_search_source.js

```diff
@@ -253,10 +336,30 @@ export class ESSearchSource extends AbstractESSource {
   getSourceTooltipContent(sourceDataRequest) {
     const featureCollection = sourceDataRequest ? sourceDataRequest.getData() : null;
     const meta = sourceDataRequest ? sourceDataRequest.getMeta() : {};
+    if (!featureCollection || !meta) {
```

Comment:
`meta` will always be defined. Can be removed from here

Reply:
Or alternatively changed to check for object emptiness

Reply:
meta will be null if `sourceDataRequest ` is not defined so this check is needed

Reply:
Ah ok. Might still be good to check for `meta` object emptiness but I'll let you make the call there